### ISSUE TITLE
should noun-noun compounds with inc/dec heads not trigger the Attachment?

### ISSUE THREAD

**Issue by bsharpataz** [clulab/eidos#389]
@MihaiSurdeanu 

I ask because since we're grounding... and some ont nodes have a sense of polarity baked in (i.e. food insecurity vs food security).

and/or we can exclude inc/dec triggers from the matching algorithm...?

**Example**
```In countries such as India , where legumes rather than animals are the preferred source of protein , these changes in the quality of food crops will accelerate the largely neglected epidemic of `` hidden hunger '' or micronutrient deficiency ( 24 ) .```

**Webapp Output**
```List(Concept, Entity) => largely neglected epidemic of `` hidden hunger '' or micronutrient deficiency ( 24 ) 
     ------------------------------ 
     Rule => gradable-lexiconner++simple-np++quantification2++Increase_ported_syntax_1_verb++Decrease_ported_syntax_2_noun 
     Type => TextBoundMention 
     ------------------------------ 
     Concept, Entity => largely neglected epidemic of `` hidden hunger '' or micronutrient deficiency ( 24 ) 
     * Attachments: Decrease(deficiency,None), Increase(accelerate,None), Quantification(largely,None) 
     ------------------------------ 

    OntologyLinkings:
        (UN/entities/human/food/food_insecurity,0.7176857984383155)
        (UN/events/human/famine,0.7118418537714329)
        (UN/entities/human/financial/economic/poverty,0.5687743265867162)
        (UN/entities/human/health/disease,0.5544075127341533)
        (UN/events/human/death,0.515939718205048)```

**Comment by MihaiSurdeanu** [comment]
Can you please explain what the problem is in the output above?

**Comment by bsharpataz** [comment]
sorry -- trying to format stuff. it's that the concept gets a Decrease bc of deficiency, and the it grounds to "food_insecurity", so we have a decrease in food_insecurity, but it's the opposite of what we want, though...

for the record, without the "deficiency" bc of hunger here it still grounds to famine and is marked as a decrease in famine (also bad) but perhaps the point is still valid?

**Comment by MihaiSurdeanu** [comment]
Maybe the solution is to *not* consider tokens for Increase/Decrease if they are the head of the phrase?

**Comment by bsharpataz** [comment]
yes, that's exactly what I was suggesting :) so you're on board then?

**Comment by MihaiSurdeanu** [comment]
yes
